install/operator/pkg/apis/syndesis/v1alpha1/syndesis_types.go

```diff
@@ -122,7 +137,20 @@ type VolumeOnlyResources struct {
 }
 
 type ServerFeatures struct {
+	// Maximum number of integrations single user can create
+	IntegrationLimit int `json:"integrationLimit,omitempty"`
+
+	// Interval for checking the state of the integrations
+	IntegrationStateCheckInterval int `json:"integrationStateCheckInterval,omitempty"`
+
+	// Whether we deploy integrations
+	DeployIntegrations bool `json:"deployIntegrations,omitempty"`
+
+	// Set repositories for maven
 	MavenRepositories map[string]string `json:"mavenRepositories,omitempty"`
+
+	// 3scale management URL
+	ManagementUrlFor3scale string `json:"managementUrlFor3scale,omitempty"`
```

Comment:
missing `ControllersIntegrationEnabled` according to the doc

Reply:
@avano I don't know which one you mean

Reply:
ah, I see it a bit earlier in the `ServerConfiguration`: https://github.com/syndesisio/syndesis/pull/7153/files#diff-9b395a0be8857358842835be96b2cd70R115

shouldn't that be moved to `ServerFeatures` as well to be consistent?

Reply:
@avano I think that property is not used anymore, instead `DeployIntegrations` is used. I am removing it

Reply:
okay